app/main/views.py

```diff
@@ -99,6 +165,43 @@ def get_service(service_id):
     return jsonify(services=jsonify_service(service))
 
 
+@main.route('/services-archive/<int:service_archive_id>', methods=['GET'])
+def get_archived_service(service_archive_id):
+    """
+    Retrieves a service from the service_archive by PK
+    :param service_archive_id:
+    :return: service
+    """
+
+    try:
+        int(service_archive_id)
+    except ValueError:
+        abort(400, "Invalid service id supplied")
+
+    service = ServiceArchive.query.filter(
+        ServiceArchive.id == service_archive_id
+    ).first_or_404()
+
+    return jsonify(services=jsonify_service(service))
```

Comment:
I think we should keep the `{"services": ...}` key in response since we're using it everywhere right now and the JSON response is the same as for normal services